packages/components/package.json

```diff
@@ -34,7 +34,7 @@
     "peerDependencies": {
         "@patternfly/react-table": ">=4.5.7",
         "@patternfly/react-core": ">=4.18.5",
-        "@patternfly/react-icons": ">=4.3.5",
+        "@patternfly/react-icons": "4.53.16",
```

Comment:
This has to use the `>=` operator. The current solution would mean that you **have to use 4.53.16**. Which is no good. We want **at least 4.53.16* or higer*